ggircs-app/app/components/FuelManagement/FuelSelectionComponent.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useCallback, useState } from "react";
+import {
+  fetchQuery,
+  GraphQLTaggedNode,
+  useRelayEnvironment,
+} from "react-relay";
+import { useRouter } from "next/router";
+import { ListGroup } from "react-bootstrap";
+import withRelayOptions from "lib/relay/withRelayOptions";
+
+interface Props {
+  queryParameter: string;
+  displayParameter: string;
+  data: {
+    edges: ReadonlyArray<{
```

Comment:
is there a generated type you can reuse here?

Reply:
I created that type to allow either a normalizedFuelType or a carbonTaxActFuelType with the optional `?` inside the edges array definition. I don't think there's a generated type I can use here.